tests/integration/test_refreshable_mat_view/test.py

```diff
@@ -198,11 +201,6 @@ def fn_setup_tables():
 )
 @pytest.mark.parametrize("with_append", [True, False])
 @pytest.mark.parametrize("empty", [True, False])
-@pytest.mark.skipif(
-    datetime.now().minute > 57,
```

Comment:
(pytest may perform this check long before the test actually runs. I didn't check how exactly it happens; probably it just does the check for all tests before running any of them, then the earlier tests may take >>3 minutes. And also it's silly to make a test ineffective 5% of the time at random.)